js/update_page.js

```diff
@@ -26,25 +26,16 @@
 
                 // Update page rendering.
                 $.ajax({
-                    url: Drupal.settings.basePath + "islandora/rest/v1/object/"
-                        + pid + "/datastream/JP2/token?" + $.param({"uses": 2}),
+                    url: Drupal.settings.basePath + "islandora/object/"
+                        + pid + "/paged_tei_seadragon_tilesource",
                     cache: false,
-                    success: function(token) {
+                    success: function(data) {
                         // Drop out here if we are not the most current request.
                         if (pid != Drupal.settings.islandora_paged_tei_seadragon.current_page) {
                             return;
                         }
                         // Update seadragon.
-                        settings.islandoraOpenSeadragon.resourceUri =
-                            location.protocol + "//" + location.host + "/" +
-                            Drupal.settings.basePath + "islandora/object/" + pid
-                            + "/datastream/JP2/view?token=" + token;
-                        tile_source = new OpenSeadragon.DjatokaTileSource(
-                            settings.islandoraOpenSeadragon.settings.djatokaServerBaseURL,
-                            settings.islandoraOpenSeadragon.resourceUri,
-                            settings.islandoraOpenSeadragon
-                        );
-                        Drupal.settings.islandora_open_seadragon_viewer.open(tile_source);
+                        Drupal.settings.islandora_open_seadragon_viewer.open(data.info);
```

Comment:
`islandora_open_seadragon_viewer` => `islandora_openseadragon_viewer`; page updater is failing with "that's not a thing"-type errors

Reply:
Update your stuff: https://github.com/Islandora/islandora_openseadragon/pull/83/files

Reply:
Actually, that's not quite it ... pokin' this

Reply:
WELL THAT'S SILLY